components/Controllers/Loans/VaultHealth.tsx

```diff
@@ -21,22 +21,25 @@ export function VaultHealth({ ltv }: VaultHealthProps) {
   const ratio = ltv / formattedMaxLTV;
 
   const indicator = useMemo(() => {
-    // Ratio, but as a number out of 10 rather than a decimal out of 1
-    const numHashes = Math.round(ratio * 10);
-    const dashes = Array(10).fill('-');
-    const hashes = Array(numHashes).fill('#');
-    return hashes.concat(dashes).join('').substring(0, 10);
+    const totalElements = 6;
+    // Ratio, but as a number out of totalElements rather than a decimal out of 1
+    const numHashes = Math.round(ratio * totalElements);
+    const empty = Array(totalElements).fill('░░');
+    const filled = Array(numHashes).fill('▓▓');
+    return filled.concat(empty).slice(0, totalElements);
   }, [ratio]);
 
   return (
     <>
       <TooltipReference {...healthTooltip}>
-        <span
+        <div
           className={
             ratio > 0.5 ? styles['indicator-danger'] : styles.indicator
           }>
-          {indicator}
-        </span>
+          {indicator.map((char, i) => (
+            <p key={i}>{char}</p>
```

Comment:
what's the reason for wrapping as a `p`?

Reply:
guess i could have done a span, is that more semantic?

Reply:
just so i'd have an element to select via css to add margin between them

Reply:
span would be preferable but not critical